src/main/resources/react/src/components/Card/inboxMessageSvg.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { InboxDiv, InboxSpan, InboxSvg } from './styledForCard';
+
+const InboxMessageSvg = (props) => {
+	return(
+		<InboxDiv>
+			<InboxSvg width="48" height="48" viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
+				<path
+					d="M40.9567 7.03256C36.4202 2.49641 30.402 0 23.9879 0C17.5739 0 11.5557 2.49641 7.01914 7.03256C-1.74942 15.8004 -2.35835 29.8351 5.50697 39.3032C4.73566 40.8558 3.49751 42.6622 1.66057 43.5653C0.777627 44.0017 0.280336 44.9455 0.432568 45.9197C0.5848 46.8939 1.33581 47.655 2.3101 47.8072C2.76679 47.8782 3.41632 47.9493 4.20792 47.9493C6.32902 47.9493 9.45485 47.452 12.6517 45.1484C16.214 47.0664 20.1111 48 23.9778 48C30.1889 48 36.339 45.5949 40.9669 40.9674C45.5034 36.4313 48 30.4135 48 24C48 17.5865 45.4932 11.5687 40.9567 7.03256ZM39.0183 39.0292C32.1679 45.8791 21.5624 47.2389 13.2201 42.3374C12.672 42.0127 11.9921 42.1142 11.5557 42.5505C11.5151 42.5708 11.4745 42.6013 11.444 42.6317C8.69369 44.7628 5.99411 45.2093 4.20792 45.2093H4.19777C6.25798 43.7074 7.55703 41.4748 8.31819 39.7395C8.43997 39.4452 8.46027 39.1408 8.38923 38.8567C8.35878 38.5827 8.24714 38.3087 8.05432 38.0854C0.625395 29.693 1.01105 16.9066 8.94741 8.97082C17.239 0.679915 30.7267 0.679915 39.0082 8.97082C47.3099 17.2617 47.3099 30.7383 39.0183 39.0292Z"
```

Comment:
If I understand right it's an image. Is it possible to extract all such images into a file with detailed information about their relations?